### PR TITLE
Recommending symfony/apache-pack when installing the project on a sha…

### PR DESCRIPTION
**Pull request by rezendus**
…red host platform

Recommending symfony/apache-pack bundle when installing the project on a shared host platform running Apache Web Server

### CODE REVIEW

File: deployment.rst [hunk 1]
```diff
@@ -187,6 +187,7 @@ setup:
 * Add/edit CRON jobs
 * :ref:`Building and minifying your assets <how-do-i-deploy-my-encore-assets>` with Webpack Encore
 * Pushing assets to a CDN
+* If you are and a shared host platform running Apache Web Server, you may install the symfony/apache-pack
```
Comment (xabbuh) [inline deployment.rst]: I think I would reword this a bit like this:

```rst
* On a shared hosting platform using the Apache web server, you may need to install the :ref:`symfony/apache-pack package <web-server-apache-mod-php>`
```

**Review by rezendus — COMMENTED** [review]
Accepted changes

**Review by rezendus — COMMENTED** [review]
Accepted changes on shared host server
